src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -339,42 +320,42 @@ void EnDns_Wait(EnDns* this, PlayState* play) {
 void EnDns_Talk(EnDns* this, PlayState* play) {
     if ((Message_GetState(&play->msgCtx) == TEXT_STATE_CHOICE) && Message_ShouldAdvance(play)) {
         switch (play->msgCtx.choiceIndex) {
-            case 0: // OK
-                switch (this->dnsItemEntry->purchaseableCheck(this)) {
-                    case 0:
+            case EN_DNS_MESSAGE_CHOICE_OK:
```

Comment:
typically we leave these as 0/1 with a comment,
but there was no suggestion to make this an enum before so it hasn't been explicitly rejected either
I would like to hear others' thoughts

me I think since the enum has only that single use, it's not needed

Reply:
fwiw, en_owl already implements an enum for this: https://github.com/zeldaret/oot/blob/9f09505d34619883748a7dab05071883281c14fd/src/overlays/actors/ovl_En_Owl/z_en_owl.c#L63

Reply:
I think the comment approach makes the most sense because the enums will never really be reuseable. The item choices are textbox specific. 
I will probably opt to remove it from this PR and eventually in the owl actor too

Reply:
Maybe they will be reuseable if item choices happen to line up (like in the owls case) but this is not the norm